rmw_opensplice_cpp/src/rmw_node.cpp

```diff
@@ -38,6 +40,10 @@ rmw_create_node(const char * name, const char * namespace_, size_t domain_id)
     RMW_SET_ERROR_MSG("namespace_ is null");
     return nullptr;
   }
+  if (security_options->enforce_security) {
```

Comment:
You must check that `security_options` is not null before accessing its members.